### DISCUSSION TITLE
Objects per zone

### DISCUSSION THREAD

**Discussion by KillahB33** [blakeblackshear/frigate#1475] [Ask A Question] [answered]
I can't seem to figure out how to isolate what objects are detected in certain zones.
Basically I want an event/clip created for cars passing onto my drive way (a zone that covers the bottom of my driveway), but I also want any other object I am tracking to get an event/clip for the rest of the camera and I am not sure how to do both.

**Comment by blakeblackshear** [comment, marked as answer]
Release 0.9.0 will allow you to specify which objects can trigger a zone. See example here: https://github.com/blakeblackshear/frigate/issues/1353#issuecomment-877497774

**Reply by KillahB33** [reply]
Too excited, that's great to hear!!
